extensions/positron-python/python_files/posit/test-requirements.txt

```diff
@@ -1,15 +1,15 @@
-duckdb<1.4.0
+duckdb
 fastcore
 geopandas; python_version < '3.14'
-haystack-ai; python_version < '3.14'
+haystack-ai
```

Comment:
Yay some of these are available now. But some still aren't so we'll keep https://github.com/posit-dev/positron/issues/9654 open